### PR TITLE
(CDAP-16353) Adding an info log entry for twill program termination status

### PR DESCRIPTION
**Pull request by wyzhang**

### CODE REVIEW

File: cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/AbstractTwillProgramController.java [hunk 1]
```diff
@@ -93,6 +93,7 @@ public void run() {
           }
           return;
         }
+        LOG.info("Twill program termination status: {}", terminationStatus.name());
```
Comment (chtyim) [inline cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/AbstractTwillProgramController.java]: Debug. Also no need to call .name

Comment (wyzhang) [reply cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/AbstractTwillProgramController.java]: Done

**Review by CuriousVini — APPROVED** [review]
lgtm code changes.
i am assuming we dont want to have log statement for null termination status?

**Comment by wyzhang** [comment]
Moved the logging up to cover when termination status is null.

**Pull request merged**